harper-core/tests/text/linters/Part-of-speech tagging.snap.yml

```diff
@@ -337,16 +337,6 @@ Message: |
 
 
 
-Lint:    WordChoice (63 priority)
-Message: |
-     148 | expensive since it enumerated all possibilities. It sometimes had to resort to
-     149 | backup methods when there were simply too many options (the Brown Corpus
```

Comment:
That's a shame, this one is actually a regression. "To" is tricky because it's a preposition but also goes with infinitive verbs.